packages/action-group/src/ActionGroup.ts

```diff
@@ -245,7 +249,10 @@ export class ActionGroup extends SizedMixin(SpectrumElement, {
                 break;
             }
             case 'multiple': {
-                this.setAttribute('role', 'group');
```

Comment:
Does this mean that `role="checkbox"` elements do not require/benefit from having a `role="group"` to contain them?

Reply:
They will receive `role="toolbar"`, which is more appropriate with roving tab index behavior, at https://github.com/adobe/spectrum-web-components/pull/3227/files/7ea146535ccb7cfdf441cdead65d1c1434b6a246#diff-b86e1ad24ea1fd0c7ca6cc46a20dd4f7e8626d26f8aa77fccbeb388cf42c9d1dR254.